src/hooks/useNotification.tsx

```diff
@@ -19,7 +21,7 @@ const useNotification = () => {
 
       console.log("Device ID from API:", deviceId);
 
-      if (deviceId) {
+      if (true) {
```

Comment:
@shreyas1434shinde it will be always tru

Reply:
@itsvick 
 comment resolved